packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -261,20 +258,28 @@ class TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
     _focusNode = FocusNode();
   }
 
+  void expandParents(T parent) {
+    if (parent?.parent?.index != -1) {
```

Comment:
nit: 
add
if (parent == null) return;
at the top of this method? This seems like it infinite loops currently unless I'm missing something which I probably am.

Reply:
Re-wrote this some point after your review.  Should be fixed.